spec.md

```diff
@@ -570,13 +559,297 @@ schema being used.
         "bullets": [
           "40 concurrent connections"
         ]
+      },
+      "schemas": {
+        "service_instance": {
+          "create": {
+            "parameters": {
+              "$ref": "osb_v2:///fakeservice/v1#BillingAccountParameters"
+            }
+          },
+          "update": {
+            "parameters": {
+              "$schema": "http://json-schema.org/draft-04/schema#",
+              "type": "object",
+              "propeties": {
+                "some-other-account": {
+                  "description": "Other fake account needed for update.",
+                  "type": "string"
+                },
+                "billing": {
+                  "$ref": "osb_v2:///fakeservice/v1#BillingAccountParameters"
+                }
+              }
+            }
+          }
+        },
+        "service_binding": {
+          "create": {
+            "parameters": {
+              "$ref": "osb_v2:///fakeservice/v1/bindings#AccountParameters"
+            }
+          }
+        }
       }
     }]
   }]
 }
 ```
 
 
+## JSON Schemas
+
+The JSON Schemas endpoint is used to define reusable definitions of JSON Schema
+Objects referenced throughout the [Catalog](#catalog-management). If a Platform
+intends to support JSON Schemas, it MUST examine the response from the catalog
+to determine if the Service Broker is using `$ref` JSON Schemas. The Service
+Broker can signal to the Platform that JSON Schema components are supported by
+using the `osb_v2:` scheme. Components are a way to allow Platforms to only
+fetch sub-schemas.
+
+The root JSON Schema document can be found by getting the JSON Schemas
+endpoint without a `:component_path`.
+
+If the `osb_v2:` scheme is used by the Service Broker, then the Platform MAY
+fetch the component JSON Schema directly using the following convention:
+
+ - Given: `osb_v2:///{:component_path}`, fetch
```

Comment:
I'd recommend clarifying the construct {:component_path}, specifically the curly braces. Quick scan through the spec indicates there is no precedent and without specifying what it means it is open to interpretation and to confusion.

Reply:
You are correct! the spec just uses :yada to mean yada is a var, removed the `{}`.